k8s/templates/job-template.yaml

```diff
@@ -63,7 +44,7 @@ metadata:
   labels:
     app: {{ template "studio.fullname" . }}
   annotations:
-    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook": post-install,pre-upgrade
```

Comment:
I'm sorry I'm not sure about this part here. Why is the job `post-install`? thank you!

Reply:
Hey! This is to help with local k8s setups. When the job is run as a `pre-install`, the local postgresql server hasn't been setup yet. So it has nothing to run on, causing this to fail!

So instead we do it during a post-install, when postgres is set up. Future upgrades will then do this as a pre-upgrade, making sure we upgrade the DB before running.